http-server/src/module.rs

```diff
@@ -45,7 +45,15 @@ impl RpcModule {
 			Box::new(move |id, params, tx, _| {
 				match callback(params) {
 					Ok(res) => send_response(id, tx, res),
-					Err(InvalidParams) => send_error(id, tx, JsonRpcErrorCode::InvalidParams.into()),
+					// TODO: this looks wonky...
+					Err(CallError::InvalidParams(InvalidParams)) => {
+						send_error(id, tx, JsonRpcErrorCode::InvalidParams.into())
+					}
```

Comment:
@niklasad1 what about this one, I am still not sure why `CallError::InvalidParams` has an `InvalidParam` field. It looks redundant here.

Reply:
You could remove it, it's redudant